doc/design/embargo-mvp.md

```diff
@@ -0,0 +1,141 @@
+# Embargo Minimum Viable Product
+Due to time constraints, we are implementing the core functionality for embargoing dandisets ASAP.
+This document is dedicated to the minimum viable product.
+A subsequent document will describe the full suite of features.
+
+
+# User experience
+
+## Web
+When creating a new dandiset, users will have checkbox to choose whether or not the dandiset should be embargoed.
+If so, the user must also specify an NIH award number (https://era.nih.gov/files/Deciphering_NIH_Application.pdf).
+The award number is an optional argument that is passed to the create dandiset endpoint.
+The award number will be added as a new `Funder` on an automatically generated Contributor Organization in the metadata.
+
+Instead of the normal `Publish` button and version browser, the DLP will instead show a `Release` button.
+Clicking it will open a confirmation modal informing the user that they are unembargoing, all their data will be publicized, there is no undoing, and that it can take some time for large dandisets.
+Confirming will lock the dandiset for the duration of the release.
+Once the release finishes, the dandiset will be like any other unpublished draft dandiset.
```

Comment:
Need to make sure the release is all or nothing, which might be a little bit tricky since it will generally involve moving a lot of data from bucket to bucket.

One approach may be to perform the full copy, validate it, and only then delete the embargoed bucket assets and finish "releasing" the dandiset.

Reply:
We should maybe also email the owner/admins when the release is done?

Reply:
since we would essentially copy from blob store to blob store, there are several s3 ways of doing that. the key is to preserve the s3 generated etag, which should also be possible. this would mean validation is simply checking the etag post copy.

Reply:
Added some text to other parts of the doc to cover all this, resolving.